.github/workflows/rust.yml

```diff
@@ -128,35 +128,12 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      - uses: taiki-e/setup-cross-toolchain-action@v1
-        with:
-          target: ${{ matrix.target }}
-
-      - uses: Swatinem/rust-cache@v2
-
-      - name: Show version information (Rust, cargo, GCC)
-        shell: bash
-        run: |
-          gcc --version || true
-          rustup -V
-          rustup toolchain list
-          cargo -V
-          rustc -V
-
-      - name: "Install cargo nextest"
-        uses: taiki-e/install-action@v2
+      - uses: prefix-dev/setup-pixi@v0.8.1
         with:
-          tool: cargo-nextest
-
-      - uses: rui314/setup-mold@v1
+          cache: true
 
       - name: "Cargo nextest"
-        run: >-
-          cargo nextest run
-          --workspace
-          --retries 2
-          --no-default-features --features rustls-tls --features slow_integration_tests
-          --status-level skip --failure-output immediate-final --no-fail-fast --final-status-level slow
+        run: pixi run test-all
```

Comment:
The compile step now takes 2 min 44 seconds, where in main it takes 1 min 29 seconds. Do you have an idea why that would be?